packages/teleterm/src/mainProcess/fixtures/mocks.ts

```diff
@@ -5,33 +5,17 @@ import { createMockFileStorage } from 'teleterm/services/fileStorage/fixtures/mo
 // Importing electron breaks the fixtures if that's done from within storybook.
 import { createConfigService } from 'teleterm/services/config/configService';
 
-const platform = 'darwin';
```

Comment:
I had to remove the hardcoded platform in order to be able to test config service for different platforms.